build-release-tools/application/make_debian_packages.py

```diff
@@ -278,12 +335,17 @@ def main():
     Exit on encountering any error.
     """
     args = parse_args(sys.argv[1:])
+    # Bintray is the offical/final place to store the deb
     bintray = Bintray(args.bintray_credential, args.bintray_subject, args.bintray_repo)
+    # Artifactory is the staging/temp place to store the deb between deb-build and post-test steps
+    artifactory = JFrogArtifactory( user_cred=(args.artifactory_username, args.artifactory_password), artifactory_loc= args.artifactory_url)
 
     checkout_repos(args.manifest_file, args.build_directory, args.force, args.jobs, git_credential=args.git_credential)
     all_repos = get_build_repos(args.build_directory)
-    package_not_exist = create_packages_filter(bintray, args.build_directory, args.is_official_release)
+    package_not_exist = create_packages_filter(bintray, artifactory, args.artifactory_repo, args.build_directory, args.is_official_release)
```

Comment:
"rackhd" must in package_not_exist 
http://10.62.59.175:8080/blue/organizations/jenkins/Master-CI-RAC-4783/detail/Master-CI-RAC-4783/382/pipeline